breathecode/marketing/tests/urls/tests_lead.py

```diff
@@ -0,0 +1,521 @@
+"""
+Test /academy/lead
+"""
+from django.utils import timezone
+from datetime import timedelta
+import re, string
+from random import choice, choices, randint
+from mixer.main import Mixer
+from unittest.mock import patch
+from django.urls.base import reverse_lazy
+from rest_framework import status
+from breathecode.tests.mocks import (
+    GOOGLE_CLOUD_PATH,
+    apply_google_cloud_client_mock,
+    apply_google_cloud_bucket_mock,
+    apply_google_cloud_blob_mock,
```

Comment:
Improve the `register_new_lead` coverage with this command `pipenv run cov breathecode.marketing`